### PR TITLE
S3 IO configuration tutorial

### PR DESCRIPTION
**Pull request by bdunogier**
A step-by-step guide to configure eZ Platform to store and service images from an Amazon S3 bucket. It contains a quick guide to S3 configuration, and step by step configuration for eZ Platform.

Most of the eZ Platform part would also apply to other cloud providers that Flysystem supports, but I haven't tested any of them yet.
### TODO
- [ ] Figure out where this should go . `doc/specifications`is clearly wrong, but I have decided to favor content over metadata :-)

### CODE REVIEW

**Comment by bdunogier** [comment]
Ping @mere-teresa and the rest of the doc team: I don't have their github usernames ;)

File: doc/specifications/io/ezplatform_aws_s3_io_configuration_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                    key: "<key>"
+                    secret: '<secret>'
+```
+
+Replacing <key> and <secret> with the ones you copied/downloaded above,
+and <region> with the amazon region that matches the one you chose when
+you created the bucket.
+
+### Flysystem S3 adapter
+eZ Platform uses Flysyste to handle content binary files. Let's create
```
Comment (wizhippo) [inline doc/specifications/io/ezplatform_aws_s3_io_configuration_tutorial.md]: Flysyste?

Comment (bdunogier) [inline doc/specifications/io/ezplatform_aws_s3_io_configuration_tutorial.md]: Missing an 'm' :)

Will fix. Comments about other týos and errors welcome.

Comment (crevillo) [inline doc/specifications/io/ezplatform_aws_s3_io_configuration_tutorial.md]: tyos => typos :stuck_out_tongue:

**Pull request closed without merging**